libarchive/archive_read_support_format_7zip.c

```diff
@@ -580,21 +603,14 @@ check_7zip_header_in_sfx(const char *p)
 }
 
 static int
-skip_sfx(struct archive_read *a, ssize_t bytes_avail)
+skip_sfx(struct archive_read *a, const ssize_t min_addr)
 {
 	const void *h;
 	const char *p, *q;
 	size_t skip, offset;
 	ssize_t bytes, window;
 
-	/*
```

Comment:
This had to be modified because find_pe_overlay and find_elf_data_sec may reed or seek the archive, hense the bytes_avail is not reliable anymore.